content.js

```diff
@@ -6,23 +6,32 @@ function compareRight(s1, s2) {
 var isWikiwand = compareRight(document.location.hostname, 'wikiwand.com');
 
 function replaceUnbalancedBraces(text) {
+  console.log('Balancing ' +  text);
   var stack = [];
   var ret = text.split('');
 
   for (var i = 0; i < text.length; ++i) {
-    if (text[i] == '(') {
+    if (text[i] == '\\') {
+      // handle special cases such as \left( or \right)
```

Comment:
Yes, these are the nasty ones. I have reported this issue to Wikiwand support quite a long time ago, but received no feedback, so I tried to fix it manually.